teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>I know this is such a small nitpicky thing to ask but could you please double-check that we always use "Dataframe" and not "Dataframe" or "dataframe"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>Do you want DataFrame? (I think you forgot to capitalize one) </p>

Reply:
Oh my, yes, DataFrame is indeed what I meant to say :)